### PR TITLE
chore: separate prettier from eslint [SFUI2-1096]

### PR DESCRIPTION
**Pull request by FRSgit**
# Related issue

Closes https://vsf.atlassian.net/browse/SFUI2-1096

# Scope of work

optimize some of npm scripts

# Screenshots of visual changes

`yarn clear` optimization:
Screenshot 2023-05-18 at 18 47 03

Vue `yarn lint:fix` before:
Screenshot 2023-05-19 at 16 06 38

Vue `yarn lint:fix` after:
Screenshot 2023-05-19 at 16 07 26

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x]  cypress tests created

### CODE REVIEW

File: apps/test/react/package.json [hunk 1]
```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "dev:not-run": "vite",
     "build-only": "vite build",
-    "lints": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "yarn eslint --max-warnings 0 && yarn prettier --check",
+    "lint:fix": "yarn eslint --fix && yarn prettier --write",
+    "eslint": "eslint --ext .tsx,.ts --report-unused-disable-directives",
+    "prettier": "prettier --loglevel warn --no-error-on-unmatched-pattern \"**/*.{ts,tsx}\"",
```
Comment (Szymon-dziewonski) [inline apps/test/react/package.json:12]: we do not need any linting here, our tests have their own linting, this is only placeholder for running tests, IMHO we should remove those so there would not be confusion

File: apps/test/react/src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
```
Comment (Szymon-dziewonski) [inline apps/test/react/src/App.tsx:1]: we do not care about this file at all

Comment (Szymon-dziewonski) [reply apps/test/react/src/App.tsx:1]: every file in testing env

File: apps/test/vue/package.json [hunk 3]
```diff
@@ -6,8 +6,10 @@
     "dev:not-run": "vite",
     "preview": "vite preview",
     "build-only": "vite build",
-    "type-check": "vue-tsc --noEmit -p tsconfig.cypress-ct.json --composite false",
-    "lints": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "lint": "yarn eslint --max-warnings 0 && yarn prettier --check",
+    "lint:fix": "yarn eslint --fix && yarn prettier --write",
+    "eslint": "eslint --ext .vue,.ts --report-unused-disable-directives",
+    "prettier": "prettier --loglevel warn --no-error-on-unmatched-pattern \"**/*.{ts,vue}\"",
```
Comment (Szymon-dziewonski) [inline apps/test/vue/package.json:12]: same here, can be removed

File: apps/test/react/src/App.tsx (hunk 2 above)
Comment (FRSgit) [reply apps/test/react/src/App.tsx:1]: Okay, so I guess I can remove these files, right?

Comment (Szymon-dziewonski) [reply apps/test/react/src/App.tsx:1]: I would say so :)

**Review by Szymon-dziewonski — APPROVED** [review]
nice one!

**Pull request merged**